### PR TITLE
Request Browser Navigation Fix

### PR DESCRIPTION
**Pull request by Noztik**

### CODE REVIEW

File: gamemodes/cinema/gamemode/modules/scoreboard/controls/cl_htmlcontrols.lua [hunk 1]
```diff
@@ -23,9 +23,10 @@ function PANEL:Init()
 
     self.BackButton.DoClick = function()
         self.BackButton:SetDisabled(true)
-        self.HTML:HTMLBack()
         self.Cur = self.Cur - 1
         self.Navigating = true
+        self.HTML:OpenURL(self.History[self.Cur])
+        self.HTML._nextUrlPoll = RealTime() + .5
```
Comment (swampservers) [inline gamemodes/cinema/gamemode/modules/scoreboard/controls/cl_htmlcontrols.lua:29]: this can probably poll faster

Comment (Noztik) [reply gamemodes/cinema/gamemode/modules/scoreboard/controls/cl_htmlcontrols.lua:29]: it has to be delayed like this cause the polling cause the pages to go back and forth which completely breaks the history system, making the delay shorter may cause bugs

Comment (swampservers) [reply gamemodes/cinema/gamemode/modules/scoreboard/controls/cl_htmlcontrols.lua:29]: Um, alright, i'll take a look later

**Comment by swampservers** [comment]
Thank you!

**Pull request merged**